### ISSUE TITLE
Dark theme type contrast

### ISSUE THREAD

**Issue by colinreedmiller** [clr-team/planning#51]
Some typographic elements have received critique re: insufficient contrast. They do pass a11y - the concern is purely visual style. Some investigation is needed to assess this concern and determine the scope of any changes.

**Comment by colinreedmiller** [comment]
A meeting with the product design and development team that reported this reveals that it is type with the <body> designation. Consequently it appears throughout applications and a change will have impact everywhere. This requires investigation and validation before proceeding. Initial report was specific to text area in form elements. A similar color appears in this products gauge component which has been designed and developed outside Clarity.
